<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WdgTest</class>
 <widget class="QWidget" name="WdgTest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1245</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QGroupBox {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #E0E0E0, stop: 1 #FFFFFF);
    border: 1px solid gray;
    border-radius: 5px;
    margin-top: 3ex; /* leave space at the top for the title */
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top left; /* position at the top center */
    padding: 0px 30px;
	border: 1px solid gray;
	border-radius: 3px;
	min-height: 25px;
  /*  background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #FF0ECE, stop: 1 #FFFFFF); */
}

QHeaderView::section {    
	background-color: rgb(227, 227, 227);
    padding-left: 4px;
    border: 1px solid lightgrey;	
	font: 10pt &quot;MS Shell Dlg 2&quot;;
}

/*QTableView {
	background-color: white;
	show-decoration-selected: 0;
	padding: 0;
	margin: 0;
}*/

/*QTableView::item:selected
{   
    color: white;
    background: #0063cd;            
}*/

QTableView::item:selected:active
{
	color: white;
	background: #0063cd; 
   /* background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #6ea1f1, stop: 1 #567dbc);*/
}

QTableView::item:selected:!active 
{
	color: white;
	/*background: #0063cd;*/
   /* background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #6b9be8, stop: 1 #577fbf);*/
   background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #ABAFE5, stop: 1 #8588B2);
}

*{outline:  none;}

</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="grpToolBox">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #E0E0E0, stop: 1 #FFFFFF);
    border: 1px solid gray;
    border-radius: 5px;
    margin-top: 6ex; /* leave space at the top for the title */
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top center; /* position at the top center */
    padding: 0px 50px;
	margin: -5px 0px 0px 0px;
  /*  background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #FF0ECE, stop: 1 #FFFFFF); */
}
</string>
     </property>
     <property name="title">
      <string>GroupBox</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item>
       <widget class="QSplitter" name="splitter_2">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="midLineWidth">
         <number>0</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="handleWidth">
         <number>3</number>
        </property>
        <widget class="QSplitter" name="splitter">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="midLineWidth">
          <number>0</number>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="handleWidth">
          <number>3</number>
         </property>
         <widget class="QTableView" name="tableView">
          <property name="styleSheet">
           <string notr="true">QHeaderView::section {    
	background-color: rgb(227, 227, 227);

    padding-left: 4px;
    border: 1px solid lightgrey;
	
	font: 10pt &quot;MS Shell Dlg 2&quot;;
}

*{outline: none; }</string>
          </property>
         </widget>
         <widget class="QTableWidget" name="tableWidget">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <column>
           <property name="text">
            <string>Name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Actual</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>New</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>-</string>
           </property>
          </column>
         </widget>
         <widget class="QGroupBox" name="grp_new">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>70</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>70</height>
           </size>
          </property>
          <property name="title">
           <string/>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="spacing">
            <number>3</number>
           </property>
           <property name="leftMargin">
            <number>3</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>3</number>
           </property>
           <property name="bottomMargin">
            <number>3</number>
           </property>
           <item>
            <layout class="QVBoxLayout" name="vLyt_addMudbusFunc">
             <item>
              <widget class="QLabel" name="label_6">
               <property name="minimumSize">
                <size>
                 <width>120</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>250</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>Fonksiyon</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="cmbAddModbusFunc">
               <property name="minimumSize">
                <size>
                 <width>120</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>250</width>
                 <height>16777215</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="vLyt_addAddress">
             <item>
              <widget class="QLabel" name="label_4">
               <property name="minimumSize">
                <size>
                 <width>90</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>90</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>Adres</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="edtAddAddress">
               <property name="minimumSize">
                <size>
                 <width>90</width>
                 <height>21</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>90</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="maxLength">
                <number>10</number>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
               <property name="clearButtonEnabled">
                <bool>false</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="vLyt_addName">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="minimumSize">
                <size>
                 <width>150</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string>İsim</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="edtAddName">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>21</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="vLyt_addVariantType">
             <item>
              <widget class="QLabel" name="label_17">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>Tür</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="cmbAddVariantType">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>16777215</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="vLyt_addPrecision">
             <item>
              <widget class="QLabel" name="label_9">
               <property name="minimumSize">
                <size>
                 <width>60</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>60</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>Hassasiyet</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="edtAddPrecision">
               <property name="minimumSize">
                <size>
                 <width>60</width>
                 <height>21</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>60</width>
                 <height>16777215</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="vLyt_addUnit">
             <item>
              <widget class="QLabel" name="label_10">
               <property name="minimumSize">
                <size>
                 <width>60</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>90</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>Birim</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="cmbAddUnit">
               <property name="minimumSize">
                <size>
                 <width>60</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>90</width>
                 <height>16777215</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item alignment="Qt::AlignBottom">
            <widget class="QPushButton" name="btnAdd">
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>80</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Ekle</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QTabWidget" name="tabWidget">
         <property name="minimumSize">
          <size>
           <width>480</width>
           <height>0</height>
          </size>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tabTable">
          <attribute name="title">
           <string>Tablo</string>
          </attribute>
         </widget>
         <widget class="QWidget" name="tabGraphic">
          <attribute name="title">
           <string>Grafik</string>
          </attribute>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
